src/web/components/elements/ImageComponent.tsx

```diff
@@ -94,13 +140,22 @@ export const ImageComponent: React.FC<{
             role={role}
             isMainMedia={isMainMedia}
         >
-            <Picture
-                sources={sources}
-                alt={element.data.alt || ''}
-                src={getFallback(element.imageSources)}
+            <div
+                className={css`
+                    position: relative;
+                `}
             >
-                {children}
-            </Picture>
+                <Picture
+                    sources={sources}
+                    alt={element.data.alt || ''}
+                    src={getFallback(element.imageSources)}
+                />
+                {starRating && (
+                    <div className={starsWrapper}>
```

Comment:
Why was it decided to have starsWrapper in the parent components instead of `starRating`? Is this CSS significantly different?

Reply:
star wrappers respond to breakpoints in different ways, I made a an excel sheet to try track these variations https://docs.google.com/spreadsheets/d/1QUa5Kh734J4saFc8ERjCYHZu10_-Hj7llNa2rr8urNg/edit?usp=sharing

Reply:
Cool, the sheet is useful, can you link to it inside the star rating component?

Reply:
sure :) but were? do i link it in all places where this logic should be?

Reply:
I think maybe just once in the component itself?